spec/services/datetime_parser_spec.rb

```diff
@@ -92,4 +92,50 @@
       end
     end
   end
+
+  describe "#issues_for" do
+    let(:date_issues) { parser.issues_for(:schedule_date) }
```

Comment:
`issues_for` is what's being tested, so it feels odd that it's in the "arrange" part of the test, rather than the "act"

Reply:
Have moved it out 👍